rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -2259,6 +2182,47 @@ public JobIdData submitFlat(@HeaderParam("sessionid") String sessionId,
     public JobIdData submitFromUrl(@HeaderParam("sessionid") String sessionId, @HeaderParam("link") String url,
             @PathParam("path") PathSegment pathSegment) throws JobCreationRestException, NotConnectedRestException,
             PermissionRestException, SubmissionClosedRestException, IOException {
+
+        return submitFromUrl(sessionId, url, pathSegment, null);
+    }
+
+    private Map<String, String> getMapWithFirstValues(MultivaluedMap<String, String> queryParameters) {
+
+        Map<String, String> parameters = new HashMap<String, String>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)